.circleci/config.yml

```diff
@@ -1702,6 +1702,22 @@ workflows:
             - oplabs-gcr-release
           requires:
             - hold
+      - docker-release:
+          name: chain-mon-docker-release
+          filters:
+            tags:
+              only: /^chain-mon\/v.*/
+            branches:
+              ignore: /.*/
+          docker_file: ops/docker/Dockerfile.packages
+          docker_name: chain-mon
+          docker_tags: <<pipeline.git.revision>>,<<pipeline.git.branch>>
+          docker_target: wd-mon
```

Comment:
should this be `wd-mon`?

Reply:
i feel like having `n` docker images that only differ with the entrypoint is annoying to maintain

Reply:
Yea this looks not good I started process of improving that here https://github.com/ethereum-optimism/optimism/pull/7840 . There should only be one image that you can pass different commands into